CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+### pg_auto_failover v1.4.2 (February 3, 2021) ###
+
+This is a bugfix release for v1.4 series
+
+#### Added
+* Implement pg_autoctl do azure commands (QA tooling). [#544]
+* pg autoctl show settings [#549]
+* Improve docker images (build, release) [#556]
```

Comment:
nit, only some of the items end with a dot at the end. This is not so easy on my eyes 👀

Reply:
Fixed.